.gitignore

```diff
@@ -126,8 +126,7 @@ dist
 .DS_Store
 
 # Generated at build time
-public/robots.txt
-public/sitemap.xml
+public/
```

Comment:
Intentional?

Reply:
Yes, running the netlify build was adding like 60 files